client/src/modules/dashboard/Events/components/EventForm.tsx

```diff
@@ -43,19 +44,33 @@ import {
 import 'react-datepicker/dist/react-datepicker.css';
 
 const EventForm: React.FC<EventFormProps> = (props) => {
-  const { onSubmit, data, loading, submitText, chapterId, loadingText } = props;
+  const {
+    onSubmit,
+    data,
+    loading,
+    submitText,
+    chapterId: initialChapterId,
+    loadingText,
+  } = props;
+  const isChaptersDropdownNeeded = initialChapterId === -1;
```

Comment:
We should change the variable name `isChaptersDropdownNeeded` because if someone doesn't know the issue and the pull request, they won't understand it.

What do you think of `associateChapters`

Reply:
Some additional context (like usage) might be needed to fully understand it, but for how it's used I think the name is fairly accurate. `associatedChapters` is for me more vague, and indicates bigger scope than this variable has.